build/iOS/TestIOSApp/UI/ViewController.m

```diff
@@ -0,0 +1,201 @@
+//
+//  ViewController.m
+//  TestIOSApp
+//
+//  Created by Raymond Cha on 3/19/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "ViewController.h"
+#include "log_bridge.h"
+#include "TestAppIOS.h"
+
+static void DebugMessageToViewCallbackFunction(const char *message, void* context);
+
+//
+// This variables are for displaying logs to a table view.
+//
+// Max logs are 100.
+const static NSInteger kMaxLogNumber = 100;
+
+typedef struct _LogInfo {
+    NSInteger count;
+    NSInteger index;
+    NSInteger postCounter;
+    NSString* logList[kMaxLogNumber];
+} LogInfo;
+
+static LogInfo s_logInfo = {
+    0,      // number of logs
+    0,      // input index
+    0,      // number of posted event.
+    {nil}   // log list
+};
+
+@interface ViewController ()
+
+- (void)addLogToListWithLogString:(NSString *)log;
+
+@property(nonatomic) NSLock* lockForLogList;
+
+@end
+
+@implementation ViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view, typically from a nib.
+
+    memset((void*)&s_logInfo, 0, sizeof(s_logInfo));
+    SetLogOutputCallback(DebugMessageToViewCallbackFunction, (__bridge void*)(self));
+    [self performSelector:@selector(processUnitTest:) withObject:[NSNumber numberWithInteger:0] afterDelay:1.0];
+}
+
+- (void)dealloc {
+    [NSObject cancelPreviousPerformRequestsWithTarget:self];
+    ResetLogOutputCallback();
+}
+
+- (NSLock *)getLogListLock {
```

Comment:
nit: code is inconsistent with bracket usage. Sometimes on new line, sometimes on the same line. Please use one style, according to language and code around you.